packages/design-system/src/components/contextMenu/stories/index.js

```diff
@@ -768,11 +768,8 @@ export const RightClickMenu = (args) => {
 export const RightClickMenuStaticValues = (args) => {
   return (
     <Grid>
-      <RightClickMenuOnShapeAndBackground>
+      <RightClickMenuOnShapeAndBackground {...args}>
```

Comment:
@BrookeGraham Requesting your review since looks like the recent file changes are from you.

`RightClickMenuOnShapeAndBackground` already includes a `ContextMenu` inside so there's no need to pass another `ContextMenu` separately. It just got broken at some point since `args` wasn't passed to it :) Should work as expected now again.